unraid_mcp/tools/rclone.py

```
@@ -1,178 +1,134 @@
+        """Manage RClone cloud storage remotes.
+
+        Actions:
+          list_remotes - List all configured remotes
+          config_form - Get config form schema (optional provider_type for specific provider)
+          create_remote - Create a new remote (requires name, provider_type, config_data)
+          delete_remote - Delete a remote (requires name, confirm=True)
         """
-        Get RClone configuration form schema for setting up new remotes.
+        all_actions = set(QUERIES) | set(MUTATIONS)
+        if action not in all_actions:
+            raise ToolError(f"Invalid action '{action}'. Must be one of: {sorted(all_actions)}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)